articles/implementing-search-view-in-android/index.md

```diff
@@ -0,0 +1,339 @@
+### A). Introduction
+When dealing with a vast amount of data, it can sometimes be strenuous to find the piece of data that you want. This problem can be solved by filtering out the necessary data with respect to the `keyword(s)` provided. This is where a `SearchView` comes in. A SearchView is an Android widget that simplifies the process of manoeuvering through a bunch of data trying to find a match. This improves the users' experience by saving their time.
+
+### B). Objectives
+In this tutorial, we will learn how to implement a SearchView in Android using a RecyclerView. The sample data used in this project will be generated explicitly for simplicity purposes.
```

Comment:
I think this sentence is somehow confusing. To me it seems like we use a recycler view to implement search view. Could you please correct this.